### ISSUE TITLE
Search includes special chars

### ISSUE THREAD

**Issue by lightonflux** [koreader/koreader#1337]
I was reading and wanted to know when something named "Marax" was introduced in the books, because i forgotten it.

So i selected Marax on the page, but instead "Marax!«" was selected. With the dictionary etc. that isn't a problem maybe because it ignores those chars, or because of fuzzy search.

But the search function searched for the exact string and was not able to find the first appearance of Marax. 

So i had to go back several pages and find a Marax surrounded by nothing but whitespace so the search would find anything useful.

TL;DR: The search (or long press selection of a word) should ignore special chars like punctuation marks.

**Comment by lightonflux** [comment]
Running 79-g76efdf7 and it does not work.

**Comment by lightonflux** [comment]
Same for version 81.
